CONTRIBUTING.md

```diff
@@ -1,13 +1,15 @@
-Help us to make OpenMS better.
-Among other things, you can
-  - implement new features (mostly C++)
+Help us to make OpenMS better and become part of the OpenMS open source community.
+
+There are many ways how you can contribute:
   - write/update documentation
-  - fix bugs
-  - ...
-  
-To incorporate your contributions, a pull request against the OpenMS 'develop' branch is probably
-the most efficient way. A core developer will review your changes and approve them (or ask for modifications).
+  - implement new algorithms, features or tools
+  - fix bugs and resolve issues
+  - many more... 
```

Comment:
As this is the message that people see when creating the first PR we might want to be more specific in what they should do and not do